### DISCUSSION TITLE
Task Working on 20/06/2022

### DISCUSSION THREAD

**Discussion by whynaidu** [Tectignis/agreerent#1] [General]
Post the pages you are working on Todays Agreerent Project

**Comment by priya8425** [comment]
sidebar

**Comment by kajal281199** [comment]
agreement.php(working on both)

**Comment by keshridivya** [comment]
consultantregistration

**Comment by shraddhha22** [comment]
delete button working admin

**Comment by AbhijeetTugave** [comment]
Working on Admin and Client Footer

**Comment by priya8425** [comment]
which options to be included under reports

**Comment by madhurianil** [comment]
I am working on agreerent header.

**Comment by snehalambavale** [comment]
change password
